spec/models/pieces/pawn_spec.rb

```diff
@@ -1,4 +1,60 @@
 require 'rails_helper'
 
 RSpec.describe Pawn, type: :model do
+  before :all do
+    @game = FactoryGirl.create(:game)
+    Piece.destroy_all
+  end
```

Comment:
We want an empty board so that it's easier to test. Problem is, `populate_board!` is always running after `create`.

Alternate way of doing this, is `stubbing` `populate_board!` so that it doesn't run.

Check this out...
```Ruby
before do
  # Note: We want the board to be empty for this test.
  allow_any_instance_of(Game).to receive(:populate_board!).and_return true
end

let(:game) { FactoryGirl.create(:game) }
```


This is one of the downsides of using callbacks in Rails. Having to stub when we don't want them to run. I'll show some even better alternatives on a team call. But this solution is good enough for now 😄 

Then since we have `game` in a let. Will need to replace `@game` with `game`